### PR TITLE
fix the bug that DL model training text is None

### PR DESCRIPTION
**Pull request by tybalex**
this PR simply fixes the bug that the training text of our DL model could be None. Add a if check to make sure there's no NoneType input.

Also add a few loggers to provide more information in logger

### CODE REVIEW

File: models/nulog/NuLogParser.py [hunk 1]
```diff
@@ -267,7 +267,11 @@ def get_test_dataloaders(self, data_tokenized, transform_to_tensor):
     def load_data(self, windows_folder_path):
         try:
             df_log = self.log_to_dataframe(windows_folder_path)
-            return [df_log.iloc[i].Content for i in range(df_log.shape[0])]
+            return [
+                df_log.iloc[i].Content
+                for i in range(df_log.shape[0])
+                if df_log.iloc[i].Content is not None
+            ]
```
Comment (sanjay920) [inline models/nulog/NuLogParser.py]: can you make this return more pandas efficient?

**Review by sanjay920 — CHANGES_REQUESTED** [review]

File: models/nulog/NuLogParser.py (hunk 1 above)
Comment (tybalex) [reply models/nulog/NuLogParser.py]: addressed.

File: models/nulog/NuLogParser.py [hunk 2]
```diff
@@ -267,7 +267,7 @@ def get_test_dataloaders(self, data_tokenized, transform_to_tensor):
     def load_data(self, windows_folder_path):
         try:
             df_log = self.log_to_dataframe(windows_folder_path)
-            return [df_log.iloc[i].Content for i in range(df_log.shape[0])]
+            return list(df_log[~df_log["Content"].isnull()]["Content"])
```
Comment (sanjay920) [inline models/nulog/NuLogParser.py]: pls change to `df_log[~df_log["Content"].isnull()]["Content"].tolist()`

**Review by sanjay920 — DISMISSED** [review]

File: models/nulog/NuLogParser.py (hunk 2 above)
Comment (tybalex) [reply models/nulog/NuLogParser.py]: done.

**Review by AmartC — APPROVED** [review]

**Pull request merged**